### DISCUSSION TITLE
Chrome is blocking all calls to app insight

### DISCUSSION THREAD

**Discussion by joer33304** [IvanJosipovic/BlazorApplicationInsights#148] [Q&A] [answered]
Works fine in firefox and edge but chrome fails:

dc.services.visualstudio.com/v2/track:1          Failed to load resource: net::ERR_BLOCKED_BY_CLIENT 

Any solution for this ?

**Comment by IvanJosipovic** [comment, marked as answer]
@joer33304, this is very likely caused by an Extension (AdBlock) which is blocking the traffic. Additionally, it could be a web traffic filter that is running on your network. In either case, there is not much that can be done from this library's perspective.
